.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: node_js
+node_js: ['10']
+os: linux
+
+# Build Packages and Storybook
+script:
+  - npm ci
+  - npm run bootstrap
+
+# Prevent Github Pages from using Jekyll
+before_deploy:
+  - touch docs/.nojekyll
+
+after_success:
+  - git config --global user.email ${GIT_COMMITTER_EMAIL}
+  - git config --global user.name stencila-ci
+  - git remote set-url origin "https://${GITHUB_TOKEN}@github.com/${TRAVIS_REPO_SLUG}.git" > /dev/null 2>&1
+  - git checkout master
+  - git checkout .
+  - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc 2> /dev/null
```

Comment:
I presume this is all necessary for `lerna publish`. If that's correct it would be useful to have a comment to that effect and to move it into `before_deploy`